lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -76,10 +78,18 @@ def self.base_model
 
     def self.model
       # Set a class-instance variable to get the appropriate model
-      @model ||= Object.const_get(/MiqAeService(.+)$/.match(name)[1].gsub(/_/, '::'))
+      @model ||= /MiqAeService(.+)$/.match(name)[1].gsub(/_/, '::').constantize
```

Comment:
Are we stuck with our current naming convention?  It feels like we're doing a lot of string manipulation to convert `_` to `::` and back.